app/controllers/api/v1/posts_controller.rb

```diff
@@ -39,6 +39,17 @@ def destroy
         end
       end
 
+      def like
```

Comment:
Why do you put custom action related to `Like` model into `Posts` controller? I'd put it into `Likes` controller into `create` and `destroy` or `update` action.